ffs/integrationtest/integration_test.go

```diff
@@ -1141,6 +1141,28 @@ func TestRepair(t *testing.T) {
 	}
 }
 
+func TestFailedJobMessage(t *testing.T) {
```

Comment:
An test (and example) to check about Deal Erros happened during a Job executiong.
In this test, we propose storing data bigger than the sector size which will definitely fail.

Now we have `DealErrors` field in the _Job_, which allows to inspect why the Job failed with a reasonable message from the Lotus node. These errors aren't _enumed_ in Lotus, only available as strings.